web/src/bootstrap_typeahead.ts

```diff
@@ -750,6 +750,7 @@ export class Typeahead<ItemType extends string | object> {
                 break;
 
             case "Escape":
+                this.select(e);
```

Comment:
This seems not conditional on the specific typeahead component -- we use this typeahead library for ~10 different components across the app. I suspect you need to add a documented option to the library for this?

Reply:
I'm still not sure I understand how this doesn't impact other typeaheads; can you explain it?

Reply:
This was addressed via making it an option to the typeahead.

I think it's probably still not quite the right code -- it seems to make `Escape` always do `select` in the compose box, regardless of whether a topic is being completed. But I think it is reasonable to merge this to unblock work stacked on top of it. 

I think we probably instead want to rework this to work like the `this.on_escape` hook, and have some way to . But I'm not sure how complex that will prove to be.